tools/remove_grouping_from_report_results.rb

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+require 'trollop'
+
+opts = Trollop.options do
+  banner "Remove extras[:grouping] from MiqReportResult#report column.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
+  opt :dry_run,    "For testing, rollback any changes when the script exits.", :short => :none, :default => false
+  opt :batch_size, "Limit memory usage by process this number of report results at a time.",    :default => 50
+  opt :count,      "Stop checking after this number of report results.",                        :default => 0
+end
+
+puts "Using options: #{opts.inspect}\n\n"
+
+if defined?(Rails)
+  puts "Warning: Rails is already loaded!  Please do not invoke using rails runner. Exiting with help text.\n\n"
+  Trollop.educate
```

Comment:
@NickLaMuro changed the `--` handling to just warn, print the help text and exit.  Let me know if this is overkill.

Reply:
Works for me.  I didn't realize what you had was standard for the `tools/` dir, so it was more of a "is this okay" out of naivety than anything else.